src/utils.ts

```diff
@@ -218,3 +219,7 @@ export function getPermitTotalPrice(
     0
   );
 }
+
+export function isValidIBAN(value: string): boolean {
+  return value.startsWith('FI') && _isValidIBAN(value);
```

Comment:
Why not to use the library inbuilt methods for checking it?
https://github.com/City-of-Helsinki/parking-permits-ui/commit/32c99d2e281404206e66f36b300a62ca2ba89398